packages/arb-token-bridge-ui/src/pages/api/screen.ts

```diff
@@ -0,0 +1,96 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { utils } from 'ethers'
+import withRetry from 'fetch-retry'
+
+function loadEnvironmentVariable(key: string): string {
```

Comment:
Should we also add tests for this? 
Or maybe open a ticket for it. This will set up a good boilerplate for our upcoming features as well. Eg. masking subgraph calls through nexjs apis.

Reply:
Sure, we can do that when we start working on routes for subgraphs